Cargo.toml

```diff
@@ -28,6 +29,11 @@ digest = "0.8.1"
 hex = "^0.3"
 blake2b_simd = "0.5.7"
 
+[dependencies.ring]
+git = "https://github.com/KZen-networks/ring.git"
+branch = "feature/p256"
+optional = true
+
```

Comment:
You probably explained it to me before, but why can't we take Ring from crate.io ?

Reply:
Because I forked and modified it (in order to extend the API of the keypair which is needed for this PR).
If we'll merge it to master (see https://github.com/KZen-networks/ring/pull/1) then we can publish KZen's repo with another name. How does that sound?

Reply:
can you give me a sense of what API extension were needed?

Reply:
Point arithmetics, serialization etc. (see PR)